pkg/controller/persistentvolumelabel/pvlcontroller.go

```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloud
```

Comment:
pvlcontroller is new.  The meat of the work is the logic from the admission controller, but there have been some minor changes to work within a controller vs the admission plugin.  The logic related to creating/starting the controller is all new